meta/rpip-status-change-checklists.md

```diff
@@ -0,0 +1,54 @@
+
+# RPIP Status Change Checklists
+
+## Draft Checklist
+
+- [ ] Ensure that the RPIP preamble is present and correctly formatted as per RPIP-1.
+  - [ ] Add RPIP number if not present.
+  - [ ] Check `title` is a few words, and not a complete sentence.
+  - [ ] Check `author` field matches convention.
+  - [ ] Check `discussions-to` field is present, and links to the Rocket Pool forum. This field should consist of a single URL.
+  - [ ] Check `status` is set to 'Draft'
+  - [ ] Check that `type` is a valid value.
+    - [ ] If the `type` is 'protocol', ensure a `category` is present. 
+  - [ ] Check that `created` is set to the day of the RPIP number assignment, in the YYYY-MM-DD format.
+- [ ] Ensure that the RPIP matches the RPIP template in terms of layout.
+- [ ] Ensure that the RPIP author is content with the draft being published on the RPIPs portal in its current state.
+
+## Finalize Checklist
+- [ ] Ensure that the RPIP Draft Checklist has been completed for this RPIP.
+- [ ] Ensure that the RPIP preamble is present and correctly formatted for a finalized RPIP.
+  - [ ] Check `status` is set to 'Final'
+- [ ] Ensure that the finalization PR is the only open pull request modifying the RPIP in the official RPIP repository.
```

Comment:
Not sure this one is quite right, there could be another one attempting to modify the RPIP (especially if living) that shouldn't block an author going forward with their draft. Maybe something like `Identify any conflict issues that would occur with active PRs if merged` would capture the same concern without implying it couldn't be finalized because of this?

Reply:
Living RPIPs have a separate checklist. This is just going Draft - >Final